cogs/PlaytesterPing.py

```diff
@@ -0,0 +1,98 @@
+import discord
+from discord.ext import commands
+import re
+import requests
+import os
+
+url = "https://api.github.com/graphql"
+githubAccessToken = os.getenv("githubAccessToken")
+
+def getLatestDiscussion():
+    headers = {
+        "Authorization": f"Bearer {githubAccessToken}"
+    }
+
+    query = """
+    query {
+        repository(owner: "R2Northstar", name: "Northstar") {
+            discussions(first: 1) {
+                edges { 
+                    node {
+                        author {
+                            login
+                            url
+                        }
+                        title
+                        number
+                        body
+                        url
+                    }
+                }
+            }
+        }
+    }
+    """
+    try:
+        response = requests.post(url, json={"query": query}, headers=headers)
+        if response.status_code == 200:
+            raw_data = response.json()
+            
+    except requests.exceptions.RequestException as err:
+        print(f"GitHub API request failed: {err}")
+        return None
+    
+    discussion_post = {
+        'author': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["author"]["login"],
+        'title': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["title"],
+        'body': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["body"],
+        'number': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["number"],
+        'url': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["url"]
+    }
+    
+    return discussion_post
+
+def getLatestReleaseName():
+    headers = {
+        "Authorization": f"Bearer {githubAccessToken}"
+    }
+
+    try:
+        response = requests.get("https://api.github.com/repos/R2Northstar/NorthstarLauncher/releases", headers=headers)
+        if response.status_code == 200:
+            reponse = response
```

Comment:
What exactly is the reasoning for making `response` read as `reponse`? (don't know the actual wording)

Especially because it doesn't seem to get used

It does work regardless, I'm just curious of the reasoning

Reply:
no idea to be honest, just removed the whole `if response.status_code == 200` part.